package.json

```diff
@@ -122,12 +122,12 @@
     "node": "^10 || ^12 || ^14"
   },
   "resolutions": {
-    "volto-slate": "4.2.1",
+    "volto-slate": "5.2.1",
```

Comment:
Check if this is correct.

Reply:
I've removed this resolution. It is also pinned in volto-clms-theme.